venv/lib/python3.12/site-packages/bson/__init__.py

```
@@ -0,0 +1,1484 @@
+    "gen_list_name",
+    "encode",
+    "decode",
+    "decode_all",
+    "decode_iter",
+    "decode_file_iter",
+    "is_valid",
+    "BSON",
+    "has_c",
+    "DatetimeConversion",
+    "DatetimeMS",
+]
```

Ruff (0.13.3)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)